README.md

```diff
@@ -2,91 +2,72 @@
 
-## Work in Progress
+For more information on the Deis workflow, please visit the main project page at https://github.com/deis/workflow.
 
-![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
+## Beta Status
 
-Deis Builder v2 is changing quickly. Your feedback and participation are more than welcome, but be
-aware that this project is considered a work in progress.
+This Deis component is currently in beta status, and we welcome your input! If you have feedback, please [submit an issue](https://github.com/deis/builder/issues). If you'd like to participate in development, please read the "Development" section below and [submit a pull request](https://github.com/deis/builder/pulls).
 
 # About
 
-This package provides a the Deis Builder, a git server to respond to `git push`es from clients. When it receives a push, it takes the following high level steps:
+The builder is primarily a git server that responds to `git push`es from clients. When it receives a push, it takes the below high level steps in order.
 
-1. Accepts the code and writes to the local file system
+1. Accepts the code and writes it to the local file system
 2. Calls `git archive` to produce a tarball (i.e. a `.tar.gz` file) on the local file system
-3. Saves the tarball according to the following rules:
-  - If the `DEIS_MINIO_SERVICE_HOST` and `DEIS_MINIO_SERVICE_PORT` environment variables exist, uses the [`mc`](https://github.com/minio/mc) client to save to the [Minio](https://github.com/minio/minio) server at `http://$DEIS_MINIO_SERVICE_HOST:$DEIS_MINIO_SERVICE_HOST`
-  - Otherwise, if the `DEIS_OUTSIDE_STORAGE_HOST` and `DEIS_OUTSIDE_STORAGE_PORT` environment variables exist, uses the [`mc`](https://github.com/minio/mc) client to save to S3 server (or server that adheres to the S3 API) at `https://$DEIS_OUTSIDE_STORAGE_HOST:$DEIS_OUTSIDE_STORAGE_PORT` (this functionality is currently waiting for merge at https://github.com/deis/builder/pull/21).
-4. Starts a builder pod according to these rules:
-  - If a `Dockerfile` is present, starts a [`dockerbuilder`](https://github.com/deis/dockerbuilder) pod, configured to download the code to build from the URL computed in the previous step (`dockerbuilder` and Dockerfile builder are not currently supported. See https://github.com/deis/dockerbuilder/pull/1 for prototype `dockerbuilder` code).
+3. Saves the tarball to centralized object storage according to the following rules:
+  - If the `DEIS_MINIO_SERVICE_HOST` and `DEIS_MINIO_SERVICE_PORT` environment variables exist, saves to the [S3 API](http://docs.aws.amazon.com/AmazonS3/latest/API/APIRest.html) compatible server at `http://$DEIS_MINIO_SERVICE_HOST:$DEIS_MINIO_SERVICE_HOST`
+  - Otherwise, if the `DEIS_OUTSIDE_STORAGE` environment variable exists, saves to `https://$DEIS_OUTSIDE_STORAGE`.
```

Comment:
We give preference to OUTSIDE storage then defaults to MINIO. later should come first then about MINIO